indexer/postcodes_matcher.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "indexer/string_slice.hpp"
+
+#include "std/cstdint.hpp"
+#include "std/string.hpp"
+
+namespace search
+{
+bool LooksLikePostcode(StringSliceBase const & slice, bool handleAsPrefix);
+/// Splits s into tokens and call LooksLikePostcode(TokenSlice) on the result.
+/// If handleAsPrefix is true returns true if some postcode starts with s.
```

Comment:
Comment about handleAsPrefix should be near the first version of LooksLikePostcode.